UPEI-Android/group-project-mindjogg
# Dev to  stage

## Purpose 
To merge Pull request #1 "dev" to "stage". 

## Review and Testing Steps

1.  Go to the files
2.  Check if there are any spelling mistakes or anything that is missing

**Note** : It is already tested and reviewed

### Notes
The templates will not work until it is in the master branch. But for safety, we do not merge any PR to master and stage branches for now. 

### Resources 
For further information about GitHub templates checkout the [GitHub Template docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates)
